adapter/to_python.py

```diff
@@ -222,68 +216,60 @@ def get_named_data_objects(self, data_object_names, kind="all"):
 
 
 class Db(object):
-    """Loads tables from a database to python
+    """Loads tables from a sqlite3 database into python
     as a dictionary of dataframes.
 
     Parameters:
 
         file_path: str
-            Path to an db file
+            Path to a sqlite db file
 
         pre_existing_keys: dictionary key index
             Keys is the previously loaded
             dictionary of dataframes
     """
 
     def __init__(self, file_path, pre_existing_keys=None):
-
-        self.db = Sql(file_path)
         self.file_path = file_path
         self.pre_existing_keys = pre_existing_keys
 
     def load(self, table_names=None):
-        """Loads tables
+        """Loads tables from a sqlite file
 
         Parameters:
 
             table_names: list
-                List of table names to load.
+                List of table names to load. Assuming the given tables always exist in the db file
                 Default: None = load all tables
+        Notes

```

Comment:
@0xd5dc is this now resolved or still pending? I don't understand the issue.

Reply:
the section was comment out because other apps don't have any handler for IO error.

Reply:
The adapter uses the logger that is used throughout our tools, please look at any of the log files for a test run. With that, unless I'm missing something, this note can be removed. If you need help with understanding how our logger works, please check in with me or someone else from the team.

Reply:
Of course, if you have ideas on improving the logging, please feel free to implement them. I would say sticking with the logger as our approach to logging is useful as we have it throughout and it allows us to create clear log files, especially the approach Evan took recently.